crates/hc_launch/src-tauri/src/launch_tauri.rs

```diff
@@ -29,7 +29,7 @@ pub fn launch_tauri(
   app_id: String,
   local_storage_dir: PathBuf,
   watch: bool,
-  passphrase: sodoken::BufRead,
+  passphrase: SharedLockedArray,
```

Comment:
Is this function exposed to be called elsewhere, rather than being called somewhere in this code?

Reply:
It is being called [here](https://github.com/holochain/hc-launch/blob/ca5e9ebc9e3e86b4297e4ced95295f08c458e035/crates/hc_launch/src-tauri/src/cli.rs#L319) with a password obtained via holochain_util [here](https://github.com/holochain/hc-launch/blob/ca5e9ebc9e3e86b4297e4ced95295f08c458e035/crates/hc_launch/src-tauri/src/cli.rs#L299), the type of which changed implicitly, hence why that type change does not show up on the calling side.